components/BottomSheet/src/MDCBottomSheetPresentationController.h

```diff
@@ -81,6 +81,11 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ When set to false, the bottom sheet controller can't be dismissed by dragging the sheet down.
```

Comment:
Shouldn't we set the default to be `YES` in the init?

Reply:
done, although the value is passed from the sheetcontroller so this may only be relevant if someone decides to use the presentation controller on its own (which isn't really recommended).